scripts/dataset-download.sh

```diff
@@ -156,7 +156,7 @@ function restore_database {
 
```

Comment:
i did this change on previous pr because of a lint warning i suppose
but now we have error
Unmatched argument at index 3: ''
on command
neo4j-admin restore --from=/data/data/.restore/aml-1.0.0 --database=aml-1.0.0 '' --to-data-directory /data/data/databases/ --to-data-tx-directory /data/data/transactions/ --move --verbose